### PR TITLE
refactor README.md contents

### PR DESCRIPTION
**Pull request by ghost**
this is a very rough edit that, for now, mainly just replaces all instances of `Node.js` with `Ayo.js` in the README file. obviously, most of the readme is broken right now, so this PR can serve as a general place to refactor the readme as far as we can.

another alternative would be to just drop the current readme altogether and draft a new one. what does everyone think?

### CODE REVIEW

**Comment by ghost** [comment]
additionally, almost all of the links point to the current website right now. how big of a priority should a new website be?

File: README.md [hunk 1]
```diff
@@ -464,7 +446,7 @@ maintaining the Node.js project.
 
 ### Release Team
 
-Node.js releases are signed with one of the following GPG keys:
+Ayo.js releases are signed with one of the following GPG keys:
```
Comment (addaleax) [inline README.md]: I don’t think this is very accurate :) How about just replacing `are signed` → `were signed` for now?

**Review by addaleax — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -464,7 +446,7 @@ maintaining the Node.js project.
 
 ### Release Team
 
-Node.js releases are signed with one of the following GPG keys:
+Ayo.js releases were signed with one of the following GPG keys:
```
Comment (varjmes) [inline README.md]: They never were. I think we should get rid of this section or say "Ayo.js is forked from Node.js, and their releases were signed with..."

File: README.md [hunk 3]
```diff
@@ -509,7 +491,7 @@ Previous releases may also have been signed with one of the following GPG keys:
 
 ### Working Groups
 
-Information on the current Node.js Working Groups can be found in the
+Information on the current Ayo.js Working Groups can be found in the
```
Comment (varjmes) [inline README.md]: No, they can't. Because we don't have any yet. :) I'd remove the section.

**Review by varjmes — COMMENTED** [review]
> additionally, almost all of the links point to the current website right now. how big of a priority should a new website be?

I'd say not a priority at all, right now.

File: README.md (hunk 2 above)
Comment (varjmes) [reply README.md]: I realise that this PR is currently a find and replace, but I think my above comment is worth doing now.

File: README.md (hunk 3 above)
Comment (varjmes) [reply README.md]: I realise that this PR is currently a find and replace, but I think my above comment is worth doing now.

File: README.md [hunk 4]
```diff
@@ -175,7 +157,7 @@ handling your report.
 
 ## Current Project Team Members
```
Comment (varjmes) [inline README.md:86]: Can we scrap this list?

**Comment by varjmes** [comment]
I'm gonna stop commenting now, but I say we Remove text from all of the subheadings we haven't dealt with (most of them) and Put a "to be written" under each. Otherwise this PR will be open forever and I'm a big fan of small iterations.

**Comment by ghost** [comment]
@varjmes done!

**Comment by varjmes** [comment]
First PR to Ayo.js, merged!

bitmoji

**Comment by addaleax** [comment]
By the way, if we want to stick to a release process similar to that of Node, it would be not mandatory but helpful if we kept its commit metadata rules to some degree

Not saying we have to :)

**Comment by ghost** [comment]
@addaleax you mean squashing PRs? i have no idea how it works in node core, to be honest

**Comment by addaleax** [comment]
When we merge PR we add metadata in the style of what you see in https://github.com/nodejs/node/commit/6a0e3b16f3f8f74635baa6e6020d7cade7fee995 (PR-URL: is the full URL, not just the #, github just doesn’t display it)

In particular, PR-URL is required by the current tooling for making releases/updating branches

**Comment by ghost** [comment]
oh, okay! should we squash/reword the commits that are now on master, then?

**Comment by addaleax** [comment]
I would *suggest* doing that until we agree that we want some other kind of format :)

(There’s even tooling like https://github.com/evanlucas/core-validate-commit for checking that data)

**Comment by aqrln** [comment]
@pup regarding the metadata, basically, what Anna said, but I'd add there is some documentation about it (https://github.com/ayojs/ayo/blob/master/COLLABORATOR_GUIDE.md#landing-pull-requests) that describes both these policies and some related Git recipes which might be helpful :)

Speaking about whether these guidelines should be changed, both leaving and omitting the `Reviewed-By` fields would work (though they may become handy since GitHub may not be there forever), but `PR-URL` definitely must be a thing, IMHO. Not only it is used by the release tooling, but I also find it extremely useful when bisecting, debugging or just exploring and tweaking code to see how it works if you can follow a URL and see additional context and discussion.

**Comment by varjmes** [comment]
Thank you @addaleax and @aqrln :)

**Comment by aqrln** [comment]
> oh, okay! should we squash/reword the commits that are now on master, then?

I'd say this repository is currently in position where doing `git rebase -i` on the new commits on `master` won't cause any harm yet, so yes, it is worth doing so. @addaleax how do you think?

**Comment by addaleax** [comment]
@aqrln Yeah, totally agree, esp with 0 open PRs. :) I can do that if nobody else is feeling like it.

**Comment by varjmes** [comment]
i would greatly appreciate someone else doing it, so I can take note :)

**Comment by addaleax** [comment]
@varjmes @pup Okay, I went ahead and squashed/reworded so the commits are fed7d02ec805955464170fd8bd77882603ecf046, 69b8f887ef115a2128a336199485cf4ad36f56a3. This would be the typical format that Node currently uses. :)

ayo

**Pull request merged**